src/obr/cli.py

```diff
@@ -133,6 +172,22 @@ def run(ctx, **kwargs):
         os.chdir(kwargs["folder"])
 
     project = OpenFOAMProject().init_project()
+
+    operations = kwargs.get("operations", "").split(",")
+    # list available operations if none are specified or given the click option or an incorrect op is given
+    if not operations:
+        print("No operation(s) specified.")
+        project.print_operations()
+        print("Syntax: obr run [-o|--operation] <operation>(,<operation>)+")
+        return
+    elif any((false_op := op) not in project.operations for op in operations):
+        print(f"Specified operation {false_op} is not a valid operation.")
+        project.print_operations()
+        return
+    elif kwargs.get("list_operations"):
+        project.print_operations()
+        return
+
```

Comment:
Since that is the same as above. Could you move it to a separate function `validate_operation -> bool`.